local-cli/runIOS/runIOS.js

```diff
@@ -82,9 +82,11 @@ function _runIOS(argv, config, resolve, reject) {
     '-derivedDataPath', 'build',
   ];
   console.log(`Building using "xcodebuild ${xcodebuildArgs.join(' ')}"`);
-  child_process.spawnSync('xcodebuild', xcodebuildArgs, {stdio: 'inherit'});
+  const xcodeBuildProcess = child_process.spawnSync('xcodebuild', xcodebuildArgs, {stdio: 'pipe'});
```

Comment:
This means we'll wait for the whole process to finish before we output anything. This will make it seem like nothing is happening.